### PR TITLE
Preprocess progress

### PR DESCRIPTION
**Pull request by chongyi0717**
solve problem 9/11

### CODE REVIEW

File: define.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
+import json
+import supporter as sp
+from enum import auto
+from strenum import StrEnum
+
```
Comment (chongyi0717) [inline define.py:5]: source class here

File: README.md [hunk 2]
```diff
@@ -1,12 +1,43 @@
...
+* 英國食品標準局
+* 香港食物安全中心 - 事故表
+* 歐洲食品安全局
+* food navigator
+
+
+
+## Run Code
+```
+python preprocess_data.py
```
Comment (Nana2929) [inline README.md:42]: Where do I specify the input and output file path?

File: preprocess_data.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+from define import *
+import os
+
+df_en = pd.read_csv('news_data/國際新聞_英文.csv')
```
Comment (Nana2929) [inline preprocess_data.py:5]: Use `argparse` or `fire` to allow argument passing.

**Review by Nana2929 — CHANGES_REQUESTED** [review]
See single-line comments.

**Pull request merged**